internal/controllers/objectsetphases/objectsetphase_controller.go

```diff
@@ -305,6 +307,7 @@ func (c *GenericObjectSetPhaseController) SetupWithManager(
 
 	return ctrl.NewControllerManagedBy(mgr).
 		For(objectSetPhase).
-		WatchesRawSource(c.dynamicCache.Source(), c.ownerStrategy.EnqueueRequestForOwner(objectSetPhase, mgr.GetRESTMapper(), false)).
-		Complete(c)
+		WatchesRawSource(c.dynamicCache.Source(),
+			c.ownerStrategy.EnqueueRequestForOwner(objectSetPhase, mgr.GetRESTMapper(), false),
+		).Complete(c)
```

Comment:
What do you think of
```suggestion
		WatchesRawSource(
			c.dynamicCache.Source(),
			c.ownerStrategy.EnqueueRequestForOwner(objectSetPhase, mgr.GetRESTMapper(), false),
		).Complete(c)
```